Documentation/design-docs/WinRT-activation.md

```diff
@@ -0,0 +1,51 @@
+# Managed WinRT Activation of .NET Core components
+
+As part of supporting a complete story for XAML Islands on .NET Core, we should provide a mechanism of activating .NET Core WinRT components. To do so, we will follow the path of the COM and IJW activations and provide a new host for activating these components in a manner similar to native WinRT components, which we will call the `winrthost`.
+
```

Comment:
Would it make sense to merge `comhost` and `winrthost`. They look very similar, both in intent and functionality

Reply:
>Would it make sense to merge `comhost` and `winrthost`.

I wouldn't push back too much here. We would need to cripple the COM side of things by inserting an empty manifest, but other than that I would support that suggestion.

Reply:
I'm 50/50 on this. I would be perfectly fine if we keep them separate. But I think we should have some explanation as to why in the doc, that's all.